test/run_tests.py

```diff
@@ -211,6 +211,45 @@ def test_inference():
     assert_dir("model_output/llik")
     # os.removedirs("model_output")
 
+def test_inference1():
+    os.chdir("test_inference1")
+
+    # build_US_setup.R
+    cmd = ["Rscript", "../../R/scripts/build_US_setup.R",
+                "-c", "config.yml",
+                "-p", "../.."]
+
+    complete = subprocess.run(cmd)
+    assert complete.returncode == 0, f"build_US_setup.R failed with code {complete.returncode}"
+
+    assert_file("data/geodata.csv")
+    assert_file("data/mobility.csv")
+
+    # full_filter.R
+    cmd = ["Rscript", "../../R/scripts/full_filter.R",
+            "-c", "config.yml",
+            "-p", "../..",
+            "-j", "1",
+            "-b", "1",
+            "-u", "test_inference1",
+            "-y", sys.executable]
+
+    complete = subprocess.run(cmd)
+
+    assert complete.returncode == 0, f"full_filter.R failed with code {complete.returncode}"
+
+    assert_file("data/test1/seeding.csv")
+    assert_file("data/us_data.csv")
+    assert_dir("model_output")
+    assert_dir("model_output/seed")
+    assert_dir("model_output/seir")
+    assert_dir("model_output/snpi")
+    assert_dir("model_output/spar")
+    assert_dir("model_output/hosp")
+    assert_dir("model_output/hpar")
+    assert_dir("model_output/llik")
```

Comment:
let's also check that a final file exists

Reply:
By final file do you mean the final folder in each hosp,spar and all folders?